### PR TITLE
Add LambdaBooks books

### PR DESCRIPTION
**Pull request by denysdovhan**
Add Ukrainian books translation by [LambdaBooks](https://github.com/LambdaBooks)

### CODE REVIEW

File: free-programming-books-ua.md [hunk 1]
```diff
@@ -24,3 +34,8 @@
 ### Python
 
 * [Пориньте у Python 3](https://uk.wikibooks.org/wiki/Пориньте_у_Python_3) - Марк Пілігрим
+
+
+### Ruby
+
+* [Маленька книга про Ruby](https://lambdabooks.github.io/thelittlebookofruby/) — Сергій Гіба (LambdaBooks)
```
Comment (eshellman) [inline free-programming-books-ua.md]: Duplicate of #2336

File: free-programming-books-ua.md [hunk 2]
```diff
@@ -16,6 +21,11 @@
 * [Програмування мовою Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
 
 
+### JavaScript
+
+* [Розуміння ECMAScript 6](http://understandinges6.denysdovhan.com/) — Денис Довгань (LambdaBooks)
```
Comment (eshellman) [inline free-programming-books-ua.md]: Travis CI linter wants you to "Remove trailing slash (http://understandinges6.denysdovhan.com)"

File: free-programming-books-ua.md (hunk 1 above)
Comment (denysdovhan) [inline free-programming-books-ua.md]: Yeah, correct. Recently, @yenel has joined @LambdaBooks team with his book.

File: free-programming-books-ua.md (hunk 2 above)
Comment (denysdovhan) [inline free-programming-books-ua.md]: Fixed

**Comment by vhf** [comment]
Thanks!

**Comment by denysdovhan** [comment]
Thank you!

**Pull request merged**